packages/url/src/index.js

```diff
@@ -7,6 +7,14 @@ const URL_REGEXP = /^(?:https?:)?\/\/\S+$/i;
 const EMAIL_REGEXP = /^(mailto:)?[a-z0-9._%+-]+@[a-z0-9][a-z0-9.-]*\.[a-z]{2,63}$/i;
 const USABLE_HREF_REGEXP = /^(?:[a-z]+:|#|\?|\.|\/)/i;
 
+/**
+ * @typedef {{[k: string]: QueryArgParsed}} QueryArgObject
+ */
+
+/**
+ * @typedef {string|string[]|QueryArgObject} QueryArgParsed
```

Comment:
This was added to more closely represent the possible return types of `getQueryArg`.

It appears that it is currently possible for the response to be an object, containing keys whose values are either `string` or `string[]` or simply a `string` or `string[]` by itself. 

This definition accounts for that. There needs to be two separate definitions because they are both cyclic and recursive.